build.gradle

```diff
@@ -88,7 +88,9 @@ if (!gradle.startParameter.isOffline()) {
 //   dot -Tsvg build/reports/visteg.dot > build_dependencies.svg
 //
 // See https://github.com/mmalohlava/gradle-visteg for further details.
-apply plugin: "cz.malohlava.visteg"
```

Comment:
This might be an alternative for visualizing task dependencies: https://github.com/dorongold/gradle-task-tree